package.json

```diff
@@ -70,12 +70,12 @@
     "@types/lodash": "^4.14.157",
     "@types/log-symbols": "^2.0.0",
     "@types/node": "^14.0.23",
-    "@typescript-eslint/eslint-plugin": "^4.0.0",
-    "@typescript-eslint/parser": "^4.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.12.0",
+    "@typescript-eslint/parser": "^4.12.0",
     "eslint": "^7.5.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prefer-arrow": "^1.2.1",
-    "jest": "^26.1.0",
+    "jest": "^26.6.3",
```

Comment:
Should maintain as much parity as possible between `jest` and `@types/jest`.  Same for other packages with types.

Reply:
Updated package typings to latest available. Resolving.